index.html

```diff
@@ -1543,6 +1538,10 @@ <h5>
                 <td>
                   A prepopulated {{DataTransfer}} object so that:
                   <ol>
+                    <li>The {{DataTransfer}} object's <a data-cite=
+                    "html/dnd.html#drag-data-store">drag data store</a> is in
```

Comment:
I wonder why this isn't exported by the HTML spec... we should probably check. To export it on the HTML side, we just need to add `data-export=""` to where this is defined. Similarly with the "read-only" definition below.